packages/flame/test/game/flame_game_test.dart

```diff
@@ -4,8 +4,598 @@ import 'package:flame/src/game/game_render_box.dart';
 import 'package:flame_test/flame_test.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
-import 'package:flutter_test/flutter_test.dart' as flutter;
 import 'package:flutter_test/flutter_test.dart';
+import 'projector_test.dart';
+
+void main() {
+  group('FlameGame', () {
+    testWithFlameGame(
+      'default viewport does not change size',
+      (game) async {
+        game.onGameResize(Vector2(100.0, 200.0));
+        expect(game.canvasSize, Vector2(100.0, 200.00));
+        expect(game.size, Vector2(100.0, 200.00));
+      },
+    );
+
+    group('components', () {
+      testWithFlameGame(
+        'Add component',
```

Comment:
These seem to sometimes be capitalized and sometimes not?

Reply:
A top-level group is the name of the class, so it's capitalized. Groups inside groups don't have any special capitalization rules, similar to individual tests.

Reply:
Not that it matters that much, but since we are organizing this now anyways, shouldn't we conform to using one of the options? Doesn't matter which one to me.